extra-network/zerotier-one/autobuild/defines

```diff
@@ -1,5 +1,5 @@
 PKGNAME="zerotier-one"
 PKGSEC=net
-PKGDEP="gcc-runtime libnatpmp miniupnpc"
+PKGDEP="gcc-runtime miniupnpc"
 BUILDDEP="ronn"
 PKGDES="A Smart Ethernet Switch for Earth"
```

Comment:
You should just capitalize the first letter of this sentence.

Another suggestion is that put `PKGDES` above `PKGDEP`.

Reply:
Also, I don't think "for Earth" makes sense to me.